sdk.js

```diff
@@ -394,6 +484,39 @@ class PaddleSDK {
 		});
 	}
 
+	/**
+	 * Get the list of all users
+	 *
+	 * @method
+	 * @param {number} [planID]
```

Comment:
Please update this parameter to reflect the implementation code (follow the format from the `constructor` jsdoc).